tests/components/switchbot/test_config_flow.py

```diff
@@ -165,7 +193,7 @@ async def test_user_form_exception(hass, switchbot_config_flow):
 
 async def test_options_flow(hass):
     """Test updating options."""
-    with patch("homeassistant.components.switchbot.PLATFORMS", []):
+    with patch("homeassistant.components.switchbot.PLATFORMS_BY_TYPE", []):
```

Comment:
We should patch the PyPi package instead of the integration itself here.

https://developers.home-assistant.io/docs/development_testing/#writing-tests-for-integrations

Reply:
Done.